meinberlin/apps/budgeting/api.py

```diff
@@ -75,6 +77,14 @@ def list(self, request, *args, **kwargs):
                 'choices': label_choices,
             }
 
+        # moderation task filter
+        moderation_task_choices = self.get_moderation_task_choices()
```

Comment:
I first tried with adding a permission to the api, but that gets more complicated in frontend, so I thought it is easiest to simply only add the filter for moderators?